Makefile

```diff
@@ -212,7 +212,12 @@ test-all-local: validate-local test-unit-local
 .PHONY: validate-local
 validate-local:
 	hack/make.sh validate-git-marks validate-gofmt validate-lint validate-vet
+
+# This invokes bin/skopeo, hence cannot be run as part of validate-local
+.PHONY: validate-docs
+validate-docs:
```

Comment:
Maybe `validate-docs-local` for consistency, and have `test-all-local` depend on it.

(I wouldn’t bother adding the `$CONTAINER_RUN` non-`local` variant.)

Reply:
… on second though, I don’t feel strongly about the `-local` suffix at all. Having a convention to always type extra is a hassle, and Chris is strongly against tests that _must_ run in a container.

This might be a good time to move towards changing the convention.

So just the `test-all-local` dependency would be nice.